userdocs/contents/boot-management.rst

```diff
@@ -318,11 +318,15 @@ load the image. This module is available in the
 ``warewulf-dracut`` subpackage, which must be installed in the
 image.
 
-With the ``warewulf-dracut`` package installed, you can build an
+With the ``warewulf-dracut`` package installed in your image, you can build an
 initramfs inside the image.
 
+Enter the image with ``wwctl image shell``
 .. code-block:: shell
+    wwctl image shell rockylinux-9
```

Comment:
Indented one space too far. Also please add blank lines around `.. code-block::`, following other examples in the file.